metadata/gc_test.go

```diff
@@ -419,8 +452,8 @@ func TestCollectibleResources(t *testing.T) {
 	}
 	removeIndex := 5
```

Comment:
Maybe you should choose other node to delete because the lease2 is expired.

Reply:
Good catch, this should bump up to 7 now